readme.md

```diff
@@ -1,5 +1,39 @@
 # Microsoft Azure IoT SDKs for Java
 
+## Upcoming Changes Notice
+
+All Azure IoT SDK users are advised to be aware of upcoming certificate changes for IoT hub and Device Provisioning Service 
+that will impact the SDK's ability to connect. In October 2022, both services will migrate away from the current 
+[Baltimore CyberTrust CA Root](https://baltimore-cybertrust-root.chain-demos.digicert.com/info/index.html) to the 
+[DigiCert Global G2 CA root](https://global-root-g2.chain-demos.digicert.com/info/index.html). There will be a 
+transition period beforehand where your IoT devices are expected to have both the Baltimore and Digicert public 
+certificates installed on your device in order to prevent connectivity issues. 
+
+For a more in depth explanation as to why the service is doing this, please see
+[this article](https://techcommunity.microsoft.com/t5/internet-of-things/azure-iot-tls-critical-changes-are-almost-here-and-why-you/ba-p/2393169).
+
```

Comment:
The lines above will be more or less ported as is to the other SDKs, but with a note about how to prepare your device by installing the DigiCert G2 certificate on your device now.